assets/onboarding/use-fullscreen.jsx

```diff
@@ -1,18 +1,27 @@
-import { useEffect } from '@wordpress/element';
+import { useLayoutEffect } from '@wordpress/element';
 
-import './fullscreen.scss';
+/**
+ * Apply fullscreen view by hiding wp-admin elements via CSS.
+ *
+ * Allows setting additional classes on the body element.
+ * Fullscreen and classes are added when the component is mounted, and removed when unmounted.
+ *
+ * @param {string[]} bodyClasses Additional classes to be set.
+ */
+export function useFullScreen( bodyClasses = [] ) {
+	bodyClasses.push( 'sensei-wp-admin-fullscreen' );
 
-export function useFullScreen() {
 	function setupGlobalStyles() {
 		toggleGlobalStyles( true );
 		return toggleGlobalStyles.bind( null, false );
 	}
 
 	function toggleGlobalStyles( enabled ) {
-		document.body.classList.toggle( 'sensei-wp-admin-fullscreen', enabled );
-		document.body.classList.toggle( 'sensei-color', enabled );
+		if ( enabled ) document.body.classList.add( ...bodyClasses );
+		else document.body.classList.remove( ...bodyClasses );
+
 		document.documentElement.classList.toggle( 'wp-toolbar', ! enabled );
 	}
 
-	useEffect( setupGlobalStyles, [] );
+	useLayoutEffect( setupGlobalStyles, [] );
```

Comment:
And here I think would be good to add the `bodyClasses` to the dependencies. BTW, we could add these eslint rules: https://reactjs.org/docs/hooks-rules.html#eslint-plugin WDYT?

Reply:
Good idea. Although calling `useFullscreen` with new classes wouldn't exactly behave as expected  right now, as the old ones wouldn't be removed. Added in 9c4ecdc.
I took a look and the React and react-hooks Lint rules are already in there, via `@wordpress/eslint-plugin/recommended`.

Reply:
Oh, excellent! \o/
I'm sorry that I didn't check it before to comment.